### DISCUSSION TITLE
How can I load multiple regions simultaneously using the tms database?

### DISCUSSION THREAD

**Discussion by mong8384** [pelicanmapping/osgearth#2116] [Support Q&A]
hello.
Currently, OSG 3.6.4 and OSGEARTH 2.10.2 are used.
When creating the TMS terrain database using osgearth_conv, it was created by dividing it into A area and B area.

After that, when the image is loaded as shown below, both regions are not applied at the same time and only the last XML file is loaded.

```
<Image driver ="tms" name="A_AREA" >
<url>Z:\DB\A\A_AREA.xml</url>
</Image >

<Image driver ="tms" name="B_AREA" >
<url>Z:\DB\B\B_AREA.xml</url>
</Image >
```

How can I load regions A and B at the same time?

**Comment by gwaldron** [comment]
Your XML looks fine. The problem may lie in the data itself, or perhaps some issue in the old version of osgEarth you are using. The current version of osgEarth is 3.3, and unfortunately, it's not possible for us to support 2.x any more.
